### ISSUE TITLE
Learn to use gradient boosted regressor 

### ISSUE THREAD

**Issue by Domlenart** [Domlenart/PoE_Trader#3]
http://scikit-learn.org/stable/modules/generated/sklearn.ensemble.GradientBoostingRegressor.html#sklearn.ensemble.GradientBoostingRegressor
